src/core/utils/chains.ts

```diff
@@ -83,6 +83,10 @@ export const getSupportedChainsWithHardhat = () => {
   );
 };
 
+export const isDefaultSupportedChain = ({ chainId }: { chainId: ChainId }) => {
+  return SUPPORTED_CHAINS.map((chain) => chain.id).includes(chainId);
+};
```

Comment:
we got a `SUPPORTED_CHAIN_IDS` already why not use that?

Reply:
i'm updatng now